app/lua53/lnodemcu.c

```diff
@@ -309,7 +309,7 @@ LUA_API int  (lua_pushlfsindex) (lua_State *L) {
  * one upvalue that must be the set to the _ENV variable when its closure is
  * created, and as such this parallels some ldo.c processing.
  */
-LUA_API int  (lua_pushlfsfunc) (lua_State *L) {
+LUA_API int lua_pushlfsfunc(lua_State *L) {
```

Comment:
The standard pattern is to add the parentheses where a define macro might be encoded to force sensible compile errors here.  Why change this?

Ditto on following cases.

Reply:
The standard pattern is to only parenthesise function names only when `typedef`'ing function pointer types. Why would these be different? If someone `#define`s a function name to something else, surely that is on them?

Reply:
I just feel that we should follow the standard Lua coding standards where practical